### DISCUSSION TITLE
Compatibility with Linux platform ?

### DISCUSSION THREAD

**Discussion by tahamunib** [SixLabors/ImageSharp#1331] [Q&A] [answered]
I want to ask about the compatibility of this api with linux ?
Is it cross-platform ?

**Comment by tocsoft** [comment, marked as answer]
ImageSharp is a solution written in fully managed code.(i.e. its completely written in C# and has no native dependencies). So yes we fully support it running cross platform running on Linux, mac, windows and theoretically should work fin in the browser too using Blazor (though Blazor isn't one of our currently target platforms for testing so your mileage may vary). Basically if it runs .net core it should run ImageSharp that was one of the design philosophies behind the development of ImageSharp in the first place.

The thing to note with running on Linux we only specifically target the .net core runtime and not mono (mostly just due to lack of having a way currently of targeting our tests against it and not because we don't think it'll work).

**Comment by tahamunib** [comment]
Awesome, Thanks.
